src/stores/socketStore.ts

```
@@ -1,7 +1,19 @@
+	WEBSOCKET_CONNECTION_TIMEOUT,
+	WEBSOCKET_RECONNECT_INITIAL_DELAY,
+	WEBSOCKET_RECONNECT_MAX_DELAY,
+	WEBSOCKET_RECONNECT_TIMEOUT,
+	WEBSOCKET_URL,
+} from '@constants/api';
+
+export type SocketConnectionStatus =
+	| 'connecting'
+	| 'reconnectWaiting'
+	| 'connected'
+	| 'disconnected';
```

ESLint

[error] Replace `|·'connecting'⏎↹|·'reconnectWaiting'⏎↹|·'connected'⏎↹` with `'connecting'·|·'reconnectWaiting'·|·'connected'·`

(prettier/prettier)